AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -96,8 +118,39 @@
 - (void)layoutDidFinish;
 
 
-/** @name Sizing */
+/** @name Layout calculation */
 
+/**
+ * @abstract Asks the node to measure a layout based on given size range.
+ *
+ * @param constrainedSize The minimum and maximum sizes the receiver should fit in.
+ *
+ * @return An ASLayout instance defining the layout of the receiver (and its children, if the box layout model is used).
+ *
+ * @discussion Though this method does not set the bounds of the view, it does have side effects--caching both the
+ * constraint and the result.
+ *
+ * @warning Subclasses must not override this; it caches results from -calculateLayoutThatFits:.  Calling this method may
+ * be expensive if result is not cached.
+ *
+ * @see [ASDisplayNode(Subclassing) calculateLayoutThatFits:]
+ */
+- (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize;
```

Comment:
I believe this is intended to be in ASDisplayNode.h, or some other category header (is there one for Layout?) - but not the Subclasses header.  I think we should move this before landing if that understanding is true.